src/Microsoft.ML.Transforms/PermutationFeatureImportance.cs

```diff
@@ -0,0 +1,224 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Float = System.Single;
+
+namespace Microsoft.ML.Transforms
+{
+    using Stopwatch = System.Diagnostics.Stopwatch;
+
+    internal sealed class PermutationFeatureImportance<TResult>
```

Comment:
>sealed [](start = 13, length = 6)

Make this a static class. There is no need to have an instance, and also you have a bug when you call the PFI method more than once.